### ISSUE TITLE
Setting to display timezone of "Generated" timestamp.

### ISSUE THREAD

**Issue by lfrankel** [doxygen/doxygen#6808]
The `HTML_TIMESTAMP         = YES` option produces text in the footer formatted like
>Generated on Fri Feb 1 2019 02:03:15

While this is useful, in our particular case we have developers in different timezones. It would be very helpful for us to be able to include the timezone in the timestamp, since that timestamp is local to the system that generated the docs.

**Comment by albert-github** [comment]
This does not only apply to `HTML_TIMESTAMP`, but also to `LATEX_TIMESTAMP`.
Rtf, and man output also the date but unconditional.
Sqllite3 outputs either date and time or only date, but also unconditionally.

There is a possibility to set one's own time by means of the environment variable `SOURCE_DATE_EPOCH` as an unsigned 64 bit integer value.

Would the later be a solution / workaround  for your problem?

**Comment by lfrankel** [comment]
> There is a possibility to set one's own time by means of the environment variable `SOURCE_DATE_EPOCH` as an unsigned 64 bit integer value.
> 
> Would the later be a solution / workaround for your problem?

Hm. Maybe. This would mean we could, say, get the time for UTC+0 and use that, and then tell all the developers that the timestamp is UTC+0. That might make more sense than telling them it's in Melbourne time. Unsure.

It's not really a big problem in any case, to be honest. I added the ticket more as something I noticed that seems to be missing, and would be nice-to-have, but it's certainly far from critical.

**Comment by lfrankel** [comment]
Yep, my boss has confirmed that I'm basically the only one who even looks at the timestamps, so this is actually a non-issue for us :)
